mod/05_email/set/type/email_template.rb

```diff
@@ -44,7 +44,7 @@ def email_config args={}
   
   [:subject, :text_message].each do |field_name|
     config[field_name] = process_email_field( field_name, args ) do |field_card|
-      field_card.contextual_content context_card, {:format=>'email_text'}, args
+      field_card.contextual_content context_card, {:format=>'email_text'}, args.merge(:content_opts=>{ :chunk_list=>:inclusion_and_link})
```

Comment:
Then I would say that this is the wrong fix for that.

Ethan just fixed up what I did a bit.  I'm not sure if this was an oversight, or to deal with other problems that came up.  The code should not be passing the url as both the text and link parameter when the chunk is rendered as a link.  That way the text expander will just render the url part and not the text in brackets because it shouldn't pass that parameter.

There should be specs that fail if the text rendering includes two copies of the url.  I'm pretty sure those got added with the link_type merge.